tools/uploadGn.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python3
```

Comment:
I propose to remove `Gn` from the script name and add a header with a license and your name.

Reply:
I use this as a "sign' which parts I had completely renewed.
If the changes are accepted I will remove this, but this implies that the original files will be overwritten.

Reply:
Done